uiowa/uiowa
# Mirroring recent updates to UIDS

Resolves #1780 
Resolves #1804 

# How to test

- Verify that `blt frontend` can run without errors.
- Initial testing should be done against the following sites to ensure there are no visual or functional regressions:
  - [ ] uiowa.edu
  - [ ] brand.uiowa.edu
  - [ ] cogscilang.grad.uiowa.edu
  - [ ] icsa.uiowa.edu
  - [ ] policy.clas.uiowa.edu

## Site Header
- Verify that site header markup is similar to: https://uids.brand.uiowa.edu/components/detail/iowa-bar--site_name_below.html.
- Test at all breakpoints. 
- Test on BrowserStack ie11 if you have access. 
- Test IOWA bar stickiness (or lack thereof). Test the scroll functionality down the page, scroll up and test the menu and search options mid-scroll and verify that there are no regressions. 

## Site Header Toggle
- In a private browser, verify that new `iowa-bar.js` is being applied to `.o-canvas__drawer` to set top position based on scroll position in browser.

## Site Footer
- Test that footer and footer logo appear the same without visual regressions (https://github.com/uiowa/uiowa/issues/1811)  
- Test footer with parent organization and without. 

## In-page menus
Verify that the in-page menu on https://uiowa.local.drupal.uiowa.edu/student-life have no visual regressions. Check a mobile breakpoint.

## Button Styles 
- Verify that #1780 has been implemented.  

## Contact form
Check that there are no visual regression on https://uiowa.local.drupal.uiowa.edu/contact

## Visual Regression Fix
- Verify that extra spacing seen in the stat grid on https://uiowa.edu/ is gone (caused by https://github.com/uiowa/uiowa/issues/1810)

## Layout Builder Style Changes
- Check that the background pattern for the events header and the stats section of the home page are in place: https://uiowa.local.drupal.uiowa.edu/
- Inspect the HTML of those areas to confirm that the background is being applied using a `bg--*` class instead of a `uids-*` class.
- Check that the stats blocks have `element--flex-center` alignment class applied to them and that they look like they do in production.
